app/assets/javascripts/app/views/notifications_view.js

```diff
@@ -7,6 +7,7 @@ app.views.Notifications = Backbone.View.extend({
   },
 
   initialize: function() {
+    alert('This should actually work for notifications dropdown');
     Diaspora.page.header.notifications.setUpNotificationPage(this);
     $(".unread-toggle .entypo").tooltip();
     app.helpers.timeago($(document));
```

Comment:
@svbergerem : I don't understand why the event  `"click .unread-toggle" : "toggleUnread"` is not binded when I had the script to the header template. Can you help me ?